client/src/components/RecommendedContainer/style.scss

```
@@ -0,0 +1,31 @@
+@import 'src/styles/package';
+
+h2{
+    font-size : 1.5rem;
+    font-weight : bold;
+    padding-left: 1rem;
+    margin : 1rem 0
+}
+
+.recommended-container{
+    width : 100%;
+    margin : 0 auto;
+    margin-top : 4rem;
+    span{
+        color: aqua;
+    }
+    .title-icon{
+        margin-right: 2px;
+        size : 2rem;
+    }
+    button{
+        width: 100%;
+        border:none;
+        font-size: 1.5rem;
```

여기 보니까 포맷팅이 안 되고 있는 것 같습니다 😭😭😭
scss 포맷팅 되는지 확인 부탁드리고, 만약 안 되면 머지한 후에 제가 포맷팅할게요!

---

아 그리고 scope가 달라지는 부분에 한줄 띄워주면 보기 더 좋을 것 같은데,
이건 제 의견이니까 적용하지 않으셔도 됩니다!

```suggestion
    }
    
    button{
        width: 100%;
        border: none;
        font-size: 1.5rem;
```

---

> 여기 보니까 포맷팅이 안 되고 있는 것 같습니다 😭😭😭
> scss 포맷팅 되는지 확인 부탁드리고, 만약 안 되면 머지한 후에 제가 포맷팅할게요!

저희가 scss는 포맷팅을 안 맞췄던 것 같은데 범수님은 포맷팅 되시나요?

---

> 아 그리고 scope가 달라지는 부분에 한줄 띄워주면 보기 더 좋을 것 같은데,
> 이건 제 의견이니까 적용하지 않으셔도 됩니다!

저렇게하는게 더 깔끔하네요! 수정해서 올리겠습니다!

---

넵넵 저는 포맷팅이 되는데, tsx 파일에서도 포맷팅이 안 되는 부분들이 보입니다 😭
vs code에서 prettier 켜져있는지 확인해주시고, formatOnSave 설정 되어 있는지 한번 봐주시면 감사하겠습니다!